lib/src/main/java/org/asamk/signal/manager/AttachmentStore.java

```diff
@@ -39,6 +39,12 @@ public File getAttachmentFile(final SignalServiceAttachmentPointer pointer) {
                 Optional.ofNullable(pointer.getContentType()));
     }
 
+    public File getAttachmentFile(AttachmentPointer pointer) {
```

Comment:
I'd prefer to have a method here that takes the attachment id from the json output (which already includes the file extension) and returns a StreamDetails, similar to how it's done in the AvatarStore.
The Manager interface would then just return the InputStream to to the Command. And the attachment command would be responsible for closing the stream.

```java
    public StreamDetails retrieveAttachment(final String id) throws IOException {
        final var attachmentFile = new File(attachmentsPath, id);
        if (!attachmentFile.exists()) {
            return null;
        }
        return Utils.createStreamDetailsFromFile(attachmentFile);
    }

Reply:
Fixed that.